phpunit/directives/utils/evaluate.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * `evaluate` function test.
+ */
+
+require_once __DIR__ . '/../../../src/directives/utils.php';
+
+/**
+ * Tests for the `evaluate` function.
+ *
+ * @group  utils
+ * @covers evaluate
+ */
+class Tests_Utils_Evaluate extends WP_UnitTestCase {
+	public function test_evaluate_function_should_access_state() {
+		// Init a simple store.
+		store(
+			array(
+				'state' => array(
+					'core' => array(
+						'a'      => 1,
+						'b'      => 2,
```

Comment:
How about throwing in a different type value (to make sure `evaluate` gets the type right)? 😄 

```suggestion
						'b'      => true,
```

(and then `assertTrue` further down).